jenkins-integration/scripts/pe_install.sh

```diff
@@ -1,34 +0,0 @@
-#!/bin/bash
```

Comment:
I deleted these two `foss_install.sh` and `pe_install.sh` scripts because they weren't used, and after working on this OPS job I'm not sure that they will be useful as they're currently written since we need to use the ruby193 shim and define a very specific set of beaker --test files to execute.

We could probably re-create these files in some form after we've written a few more jobs and could see what/how to extract the installation.  For the time being, the code in `setup_master.sh` is a better template to follow for any new JJB jobs we define.